tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_deposit_request.py

```diff
@@ -18,6 +24,7 @@ def test_new_deposit_under_max(spec, state):
     deposit_request = prepare_deposit_request(spec, validator_index, amount, signed=True)
 
     yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
+    yield from run_process_pending_deposits(spec, state)
```

Comment:
I would rather leave these test cases as they are and remove most of them (will think on which to keep actually), and copy and paste all of these scenarios into `test_apply_pending_deposits` to see that the logic of applying a deposit did not change